docs/about/index.md

```diff
@@ -2,4 +2,4 @@
 
 In these pages, we try to describe both what Perlang _is_ and what it aims to _become_. We try to make the distinction betwen these clear, when relevant.
 
-If you just want to jump straight into some examples so you can get a feeling of what it looks like, head straight to the [learn](learn) subpage.
+If you just want to jump straight into some examples so you can get a feeling of what it looks like, head straight to the [learn](../learn/quickstart/index.md) section.
```

Comment:
These kind of changes are a bit unpleasant since they will include `index.html` in the URLs (which I think looks very ugly, TBH), but it unfortunately plays better with DocFx. (It emits a warning for links that do not specifically point at a particular page - if anyone reading this knows how to work around this, patches/suggestions are highly appreciated!)